tests/test_p2p_daemon.py

```diff
@@ -36,7 +36,7 @@ async def test_daemon_killed_on_del():
 
 @pytest.mark.asyncio
 async def test_startup_error_message():
-    with pytest.raises(P2PDaemonError, match=r"Failed to connect to bootstrap peers"):
+    with pytest.raises(P2PDaemonError, match=r"failed to connect to bootstrap peers"):
```

Comment:
```suggestion
    with pytest.raises(P2PDaemonError, match=r"(?i)Failed to connect to bootstrap peers"):
```

`(?i)` will make the regex case-insensitive (it seems like we have both upper- and lowercase in different versions of the daemon).